fairscale/nn/data_parallel/sharded_ddp.py

```diff
@@ -71,6 +78,14 @@ class ShardedDataParallel(nn.Module):
         handled. In that case ShardedDDP will raise an exception and suggest to either remove the unused parameters from your model
         (https://pytorch.org/docs/stable/generated/torch.nn.parallel.DistributedDataParallel.html?highlight=unused_parameters is helpful)
         or set `reduce_buffer_size` to 0
+
+    .. warning:
```

Comment:
tentatively explain the two options:
- auto detect (default)
- manual + explicit refresh() call